nrf52-hal-common/src/spim.rs

```diff
@@ -55,6 +61,92 @@ impl_spim_ext!(
 /// - The over-read character is hardcoded to `0`.
 pub struct Spim<T>(T);
 
+impl<T> Transfer<u8> for Spim<T> where T: SpimExt
+{
+   type Error = Error;
+
+    fn transfer<'w>(&mut self, words: &'w mut [u8]) -> Result<&'w [u8], Error> {
+        // TODO: some targets have a maxcnt whose size is larger
```

Comment:
Is this comment still true? I think the feature flags that you do on lines 48-51 means this comment can be removed

Reply:
Thanks, not only the comment but also the code it commented should be removed!